test/suites/ACID.MATLAB.8.3.md5

```diff
@@ -40,7 +40,7 @@ legendsubplots : 528e57cc9c7858f9c941bd64733b2f12
 linesWithOutliers : 4976c3f7b1e8e8cd32f7e17e34fcc99f
 logbaseline : 59ff71a8a8125053e2ddac4dd3070e5a
```

Comment:
Different MATLAB/octave versions behave differently with respect to some axis/figure properties. As such the code path inside m2t differs and consequently different output is generated. We have an ongoing effort to unify things but it is unlikely to ever achieve that always the same tex output is generated.